test/verify/check-machines-dbus

```diff
@@ -562,9 +562,12 @@ class TestMachinesDBus(machineslib.TestMachines):
 
         args = self.startVm("subVmTest1")
 
-        m.execute("virsh attach-disk --domain subVmTest1 --source /mnt/vm_one/mydiskofpoolone_1 --target vdc --targetbus virtio")
-        m.execute("virsh attach-disk --domain subVmTest1 --source /mnt/vm_one/mydiskofpoolone_2 --target vdd --targetbus virtio --persistent")
-        m.execute("virsh attach-disk --domain subVmTest1 --source /mnt/vm_one/mydiskofpoolone_3 --target vde --targetbus virtio --persistent")
+        d1 = os.path.join(p1, "mydiskofpoolone_1")
```

Comment:
Just a hint for simplification: These variables aren't being used anywhere else, so you could just write `{0}/mydiskofpoolone_1` in the commands below and `.format(p1)`.